termtosvg/__main__.py

```diff
@@ -150,6 +156,11 @@ def main(args=None, input_fileno=None, output_fileno=None):
         else:
             svg_filename = args.output_file
 
+        if args.pp is False:
```

Comment:
Why not do this check & assignment outside the if/elif/else block? Would be a bit more DRY.

Reply:
It was done this way before, I didn't want to make a too big change.
We probably shoot for reversing the entire block , this way ? 
```python
    if command == 'record':
        #[lotta code]
    else:
        #parsing options accepted for "[nocommand]" and "render" command
        if args.pp is False:
            pp = configuration['GLOBAL']['playpause']
        else:
            pp = True

        if args.font is None:
            font = configuration['GLOBAL']['font']
        else:
            font = args.font

        fallback_theme_name = configuration['GLOBAL']['theme']
        fallback_theme = configuration[fallback_theme_name]
        cli_theme = configuration.get(args.theme)
        if command == 'render':
            [RENDER CODE]
        else:
            [NO_COMM CODE]
```
this would avoid several duplicate lines, is this what you had in mind ?